php-7-ci/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM php:7.1
+RUN apt-get update
+
+RUN apt-get install -y wget
+RUN wget https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer -O - -q | php -- --quiet && \
+    php composer-setup.php && \
+    php -r "unlink('composer-setup.php');" && \
+    mv composer.phar /usr/local/bin/composer
+
+RUN wget https://github.com/generationtux/par/releases/download/v0.0.2/par_linux_amd64
+RUN mv par_linux_amd64 par
+RUN chmod +x ./par
+RUN mv ./par /usr/local/bin/
```

Comment:
We can combine all of these run instructions to shrink the image size.

```Dockerfile
RUN apt-get update && \
    apt-get install -y wget && \
    ...
```

We should also remove all of the install packages after building the image.